news/2018-08-18-aspire-2018-begins.md

```diff
@@ -0,0 +1,68 @@
+
+This year, the song choice is **Acid Rain** from **Culprate**. 
+
+Hearkening back to the earlier song choices from the contest's roots, Acid Rain is all about variation and exploration, jumping from one musical theme to the next.
+
+Featuring heavy syncopation and rapidly evolving melodies and synthlines all linked back to central motifs, Acid Rain provides a huge amount of space for mappers to lay out their creative enterprise - but beware, for that same space also allows competing mappers to flex their creative muscles!
+
+The true test of this track is to simply keep up with it. Can you keep your map continually evolving to match the sound, or will you get left behind?
+
+We'll see.
+
+The rules for this contest are as follows:
+
+* This contest is for **osu!** (standard) mode only.
+* **The osu! Ranking Criteria DOES NOT APPLY to this contest.**
+* **You may submit ONE single difficulty to the contest**. This may be of any difficulty tier that you wish, though Aspire contests typically attract Expert (5.25+) and above.
+* **The map itself must be able to be passed, with mods or otherwise.** Maps which autofail (defined as immediately failing at the end of the map when played by Auto) will be disqualified, no questions asked.
+* **Any entries with additional resources (hitsounds, storyboards, etc) must be submitted in .osz format.** There are no size restrictions this time around, but if your .osz file does not get accepted by the contest system for being too large, PM [Ephemeral](http://osu.ppy.sh/u/102335) via the forums for assistance.
+* **The drain time of your entry must be at least 70% of the song's total length.** Basically, you have to map at LEAST 70% of the song. I recommend you map all of it.
+
+All entries will be judged by a panel of expert community members selected from across the beatmapping scene and reduced to roughly 12 in total before being put up for community voting after a stream feature. The winning sets will then be decided based on a combination of double-blind community voting and blind scoring from the panelists.
+
+Given the rising presence of "spectacle" entries into the Aspire contests, I will also be introducing a new scoring category revolving around visual impression, aka, spectacle. 
+
+Each set will be awarded a visual spectacle score by the panelist judges that is completely separate from all other scoring, and the set with the highest spectacle score at the end of judging will receive a special prize. Anything that influences how the map visually appears (aka, storyboarding, art, hitsounds) will count towards this score.
+
+Visuals will remain an important aspect of Aspire, but this new scoring should allow for more focus on the beatmapping aspect of creativity for those not so visually inclined. The super-high achievers among you may also note that the winning entry can also conceivably score 1st place in the normal criteria, and also win the spectacle voting for two prizes in one.
```

Comment:
those not so visually inclined -> for those that are less concerned with aesthetics (or something)
current wording sounds like the mappers are blind or something